src/Juniper.jl

```diff
@@ -4,6 +4,16 @@ using MathProgBase
 using JuMP
```

Comment:
You probably want to use `import JuMP` and prefix all JuMP functions with `JuMP.`.
http://www.juliaopt.org/JuMP.jl/latest/style/#using-vs.-import-1

Reply:
now `import JuMP` and using macros like `@variable,@constraint,@objective`